app/controllers/posts_controller.rb

```diff
@@ -3,10 +3,26 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: params[:user_id]).order(id: :asc)
-    @posts = @posts.paginate(page: params[:page], per_page: 2)
+    @posts = @posts.paginate(page: params[:page], per_page: 10)
   end
 
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
```

Comment:
- [ ] Kindly fix this error:

![image](https://github.com/GhostEsso/Blog_App/assets/24703171/1841fd2f-4e82-44de-ba85-0b12ef6d736a)
I get this error when I try to create a new post.